Artifacts.toml

```diff
@@ -0,0 +1,9759 @@
+[[tzdata_1996l]]
+arch = "x86_64"
+git-tree-sha1 = "835e88b3b0a560e79dfc0a2a6c350adeaa4ef907"
+lazy = true
+os = "windows"
```

Comment:
The tzdata artifacts aren't OS specific so this can be removed safely and this file can be 4x smaller!

Reply:
Yes. The `git-tree-sha1` is platform specific, which confused me, see: https://github.com/JuliaLang/Pkg.jl/issues/1914
but there I learned it's not controlled on Windows, so I'll make it platform independent and use the linux ones.

Reply:
Interesting. Thanks for providing that reference